unit-tests/src/test/java/org/eclipse/collections/impl/list/IntervalTest.java

```diff
@@ -417,7 +417,7 @@ public void invalidIntervals()
     public void toFastList()
```

Comment:
Maybe change this test name `toList`, since it is not testing `toFastList`. We could assert that the type is `FastList`, but not sure if that is necessary.